package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "qcheck-dev",
+    "esy": {
+      "build": "refmterr dune build -p qcheck-rely",
+      "install": [
+        "esy-installer qcheck-rely.install"
+      ]
+    },
+    "scripts": {
+        "example-rely": "dune exec example/rely/QCheck_rely_test.exe"
+    },
+    "dependencies": {
+      "@opam/dune": "*",
+      "@opam/qcheck": "*",
```

Comment:
should the version be `^0.9` there?

Reply:
```^0.9``` isn't working because it's getting resolved to ```@opam/qcheck@>=opam:0.9.0 <opam:0.10.``` and 0.9 < 0.9.0. I'm going with ```>= 0.9 < 0.10``` instead. I think this should be good?